lib/compress/native-objects.js

```diff
@@ -174,6 +195,9 @@ export const is_pure_native_value = make_nested_lookup({
         "SQRT1_2",
         "SQRT2",
     ],
+    String: [
+        "raw",
+    ],
```

Comment:
According to the comment this object is all about native numeric values, and String.raw is a function.

```suggestion
```